eng/BrewFile

```diff
@@ -0,0 +1,8 @@
+brew "autoconf"
+brew "automake"
+brew "cmake"
+brew "icu4c"
+brew "libtool"
+brew "openssl@1.1"
```

Comment:
I see we specified 1.1 here as well as added other packages.  What's the minimal fix here, since this is servicing.

Reply:
removed the extras to minimal change.